generators/bdk/python/templates/bot-app/activities.py.ejs

```diff
@@ -0,0 +1,39 @@
+from symphony.bdk.core.activity.command import CommandActivity, CommandContext
+from symphony.bdk.core.activity.user_joined_room import UserJoinedRoomActivity, UserJoinedRoomContext
+from symphony.bdk.core.service.message.message_service import MessageService
+from symphony.bdk.core.service.user.user_service import UserService
+
+
+class EchoCommandActivity(CommandActivity):
```

Comment:
would it be simpler as a SlashCommandActivity ?

Reply:
I wanted to exemplify the use of a CommandActivity (not a SlashCommand)